samples/msal-node-samples/client-credentials/index.js

```diff
@@ -29,8 +29,8 @@ const cca = new msal.ConfidentialClientApplication(config);
 // The scope is always in the format "<resource>/.default"
 const clientCredentialRequest = {
     scopes: ["https://graph.microsoft.com/.default"],
-    azureRegion: "westus2",
-    skipCache: true, 
+    azureRegion: "REGION_NAME", // (optional) specify the region you will deploy your application to here (e.g. "westus2")
```

Comment:
@sameerag added this back and pointed to docs in readme

Reply:
Makes sense to me.